fhir-install/src/main/docker/ibm-fhir-schematool/build.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+# ----------------------------------------------------------------------------
+# (C) Copyright IBM Corp. 2020
+#
+# SPDX-License-Identifier: Apache-2.0
+# ----------------------------------------------------------------------------
+
+set -e -o pipefail
+
+##############################################################################
+
+docker build --build-arg FHIR_VERSION=4.5.0 -t ibm-fhir-schematool:latest .
```

Comment:
we'll need to remember to keep this version reference up-to-date.  maybe use your curl trick to get the latest version from the github api?

Reply:
I've just added automation to it.